jobs/quicksuggest2bq/quicksuggest2bq/main.py

```diff
@@ -33,24 +33,24 @@ class KintoSuggestion:
     url: str
 
 
-def download_suggestions(client: kinto_http.Client) -> Dict[int, KintoSuggestion]:
+def download_suggestions(client: kinto_http.Client) -> Iterator[KintoSuggestion]:
```

Comment:
Note that we haven't figured out what IDs to be used for Wikipedia suggestions for Firefox Suggest Offline Expansion, it's currently [hardcoded](https://github.com/mozilla-services/quicksuggest-rs/blob/main/wikipedia/make_suggestions.py#L134) as 0, so I need to change this from Dict to Iterator.